assets/js/apitool-methods-ws.js

```diff
@@ -211,6 +211,17 @@ Request('ledger_entry - DirectoryNode', {
   }
 })
 
+Request('ledger_entry - NFT Page', {
+  description: "Returns an NFT Page object in its raw ledger format.",
+  link: "ledger_entry.html#get_nft_page",
```

Comment:
The link checker doesn't check the WS tool page because it's not smart enough to run all the JavaScript first, but I think this link is broken. Based on the changes to `ledger_entry.md`, the correct link is:

```suggestion
  link: "ledger_entry.html#get-nft-page",
```